frontend/components/customizer/customizer.tsx

```diff
@@ -15,6 +15,9 @@ function getBaseUrlFromWindowLocation(): string {
   // Default to the current hostname for when there is no `BASE_URL` set
   // at build time (as in most PaaS deploys).
   const { protocol, hostname } = window.location
```

Comment:
Could you paste in some of the explanation from https://github.com/badges/shields/issues/5014#issuecomment-729154310 ?